Changelog.md

```diff
@@ -21,6 +21,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Removed experimental AMD extensions (#607)
 - Removed misnamed, deprecated `debug_utils_set_object_name()` and `debug_utils_set_object_tag()` entirely, use `set_debug_utils_object_name()` and `set_debug_utils_object_tag()` instead (#661)
 
+### Fixed
+
+- `VK_KHR_draw_indirect_count`: use `cmd_draw_indirect_count_khr` instead of `cmd_draw_indexed_indirect_count_khr` for non-indexed draw call (#695)
```

Comment:
Gah, I should start cleaning up our `<subsystem>:` prefix in the changelog, it's all over the place everywhere :)

Reply:
Want me to change it to something else? I made it look a bit more consistent with the others.

Reply:
I think I'll fix `khr::RayTracingPipeline` separately, or feel free to replace that with `VK_KHR_ray_tracing_pipeline`. We don't have many **prefixes** like this, but most changelog entries are referring to an extension by its name rather than the file path or Rust module path anyway.

Reply:
In that sense removing the colon (`:`) and making this a complete sentence is fine by me as well (maybe even has my preference now).

Reply:
Yeah,  `khr::RayTracingPipeline` is the odd one out of the bunch.

I think it's worth doing as a separate PR though.
I'd touch a bunch of changelog entries that will end up pointing to this unrelated PR.

Reply:
Feel free to submit one (and feel free to make this one a sentence as part of this PR), otherwise I'll try to fix them up with issuing the next release, if I don't forget.

Reply:
> In that sense removing the colon (`:`) and making this a complete sentence is fine by me as well (maybe even has my preference now).

@teoxoy what's your thought, stick to `` `VK_KHR_draw_indirect_count`: `` or turn this into a more naturally flowing sentence without that prefix?

Reply:
Tbh, I think it's fine as it is. I'd imagine most people using ash would know what to look for and the ext name makes it easy to spot.

Reply:
Ack, I'll refactor these all at once later then.